pennylane/drawer/_add_obj.py

```diff
@@ -155,6 +176,21 @@ def _add_op(obj: Operator, layer_str, config, tape_cache=None, skip_grouping_sym
     return layer_str
 
 
+@_add_obj.register
+def _add_global_phase(
+    obj: GlobalPhase, layer_str, config, tape_cache=None, skip_grouping_symbols=False
+):
+    n_wires = len(config.wire_map)
+    if not skip_grouping_symbols:
+        layer_str = _add_grouping_symbols(list(range(n_wires)), layer_str, config)
```

Comment:
```suggestion
        layer_str = _add_grouping_symbols(list(wire_map.keys()), layer_str, config)
```
Since they get mapped inside `_add_grouping_symbols`.

Reply:
Ah yes, thanks!